server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -476,6 +501,9 @@ public void writeTo(StreamOutput out) throws IOException {
             if (out.getVersion().onOrAfter(REPOSITORY_ID_INTRODUCED_VERSION)) {
                 out.writeLong(entry.repositoryStateId);
             }
+            if (out.getVersion().onOrAfter(Version.V_6_7_0)) {
```

Comment:
We will have to mute BwC tests before merging this and then adjust this version to `6.7` in `master` afterwards, currently we have a `7.0.0` here in master.